src/Info.hs

```diff
@@ -123,9 +123,11 @@ getSrcSpan (GHC.RealSrcSpan spn) =
          , GHC.srcSpanEndCol spn)
 getSrcSpan _ = Nothing
 
-getTypeLHsBind :: GHC.TypecheckedModule -> GHC.LHsBind TypecheckI -> GHC.Ghc (Maybe (GHC.SrcSpan, GHC.Type))
+getTypeLHsBind :: GHC.TypecheckedModule -> GHC.LHsBind TypecheckI
+    -> GHC.Ghc (Maybe (GHC.SrcSpan, GHC.Type))
 #if __GLASGOW_HASKELL__ >= 708
-getTypeLHsBind _ (GHC.L spn GHC.FunBind{GHC.fun_matches = grp}) = return $ Just (spn, HsExpr.mg_res_ty grp)
+getTypeLHsBind _ (GHC.L spn GHC.FunBind{GHC.fun_matches = grp}) =
+    return $ Just (spn, HsExpr.mg_res_ty $ HsExpr.mg_ext grp)
```

Comment:
HsExpr.mg_ext seems to not be available in older versions of ghc used in some stackage snapshots (see travis for details)

Reply:
Hmm ... not sure how to fix this.  Perhaps the 708 should be increased on line 128?